Helper/Requests.php

```diff
@@ -357,6 +357,17 @@ public function buildThreeDS2Data($additionalData, $storeId, $request = [])
         return $request;
     }
 
+    /**
+     * @param array $request
+     * @return array
+     */
+    public function buildRedirectData($storeId, $request = [])
+    {
+        $request['returnUrl'] = rtrim($this->adyenHelper->getOrigin($storeId), '/') .
+            '/adyen/process/redirect';
```

Comment:
@acampos1916 shouldn't this be the transparent controller?

Reply:
@cyattilakiss will check!

Reply:
The transparent controller handles POST parameters, and this returnUrl is used when the 3DS1 behavior returns a GET redirectResult. This redirectResult is handled in `/adyen/process/redirect`